src/host.js

```diff
@@ -7,7 +7,7 @@ import {hostSyncTitle} from './plugins/sync-title';
 var originRe = /^(http:\/\/|https:\/\/)[^\/]+/i;
 
 export default class Host extends Port {
-	constructor(elementProvider, src, options) {
+	constructor(elementProvider, src, options, height, frameId) {
```

Comment:
We should use the generic options object for the height and frameId parameters. Let's call them `height` and `id`.